js/main.js

```diff
@@ -30,6 +30,85 @@ function handleSubmit(event) {
   $photo.setAttribute('src', 'images/placeholder-image-square.jpg');
 
   $form.reset();
+
+  $ul.prepend(renderEntry(formValues));
+  viewSwap($entryView);
+
+  toggleNoEntries();
 }
 
 $form.addEventListener('submit', handleSubmit);
+
+function renderEntry(entry) {
+  const $li = document.createElement('li');
+  $li.setAttribute('class', 'row');
+
+  const $picDiv = document.createElement('div');
+  $picDiv.setAttribute('class', 'column-half');
+
+  const $entryImage = document.createElement('img');
+  $entryImage.setAttribute('src', entry.photo);
+
+  const $chDiv = document.createElement('div');
+  $chDiv.setAttribute('class', 'column-half');
+
+  const $entryTitle = document.createElement('p');
+  $entryTitle.textContent = entry.title;
+
+  const $entryNote = document.createElement('p');
+  $entryNote.textContent = entry.notes;
+
+  $li.appendChild($picDiv);
+  $picDiv.appendChild($entryImage);
+  $li.appendChild($chDiv);
+  $chDiv.appendChild($entryTitle);
+  $chDiv.appendChild($entryNote);
+
+  return $li;
+}
+
+const $ul = document.querySelector('.ul');
+
+document.addEventListener('DOMContentLoaded', (event) => {
+  for (let i = 0; i < data.entries.length; i++) {
+    const $newEntry = renderEntry(data.entries[i]);
+    $ul.appendChild($newEntry);
+  }
+  viewSwap('entries');
```

Comment:
You shouldn't be hard coding the string you pass in, you should be passing in `data.view`